src/Controller/Component/AuthComponent.php

```diff
@@ -242,8 +242,7 @@ public function initialize(array $config)
     {
         $controller = $this->_registry->getController();
         $this->eventManager($controller->eventManager());
-        $this->request = $controller->request;
```

Comment:
I don't think it will, since `$this->request` is already set in `Component::_construct()` this re assignment is redundant.